src/components/Simulation/SimulationBlock/Table/index.js

```diff
@@ -72,7 +72,13 @@ const cellTooltip = (cell, t) => {
     return t("tooltips.cell.output");
   }
   if (isTopic(cell.value)) {
-    return cell.value.value;
+    return (
+      <div>
+        {t("topic.code")} : {cell.value.key}
```

Comment:
(En Francais it's a space before :, in English there isn't, but I don't think we should focus on that right away. Mainly because  I don't have a good answer on how to handle those)

Reply:
since it's a field value separator, are you sure this really applies, not really a sentence.
`eg: blabla is really blabla`
you want me to put `:` in the translation file ?

Reply:
No, I think this is the right way to handle it. If we ever get a request for it, we'll have to use a helper. Don't think a translation is the right way to solve it.